main/src/ca/uwaterloo/flix/language/ast/Scheme.scala

```diff
@@ -18,7 +18,9 @@ package ca.uwaterloo.flix.language.ast
 
```

Comment:
I have not reviewed this class, but I will once I have re-read the paper and its implementation has become a bit more "stable".